webrender/res/cs_border_solid.glsl

```diff
@@ -186,29 +168,6 @@ void main(void) {
     bool in_clip_region = all(lessThan(vClipCenter_Sign.zw * clip_relative_pos, vec2(0.0)));
     float d = -1.0;
```

Comment:
nit: later on, we don't need `d = max(d, d_radii)`

Reply:
Nice catch, thanks :)